lib/Crypt/MatrixSSL3.pm

```diff
@@ -772,12 +771,12 @@ Parameters:
 
 =item $server_index and $index
 
-Are the same as refresh_OCSP_staple above, but $index take the return
-value of the first $ssl->set_SCT_buffer(...) call.
+Are the same as refresh_OCSP_staple above.
 
 =item $SCT_params
 
 Perl scalar contains a file name with prepared extension data.
+or
```

Comment:
This won't be formatted correctly because POD doesn't keep non-indented lines as is, they'll be reformatted to single paragraph.